pkg/gui/gui.go

```diff
@@ -454,7 +454,13 @@ func (gui *Gui) onUserConfigLoaded() error {
 	} else if userConfig.Gui.ShowIcons {
 		icons.SetNerdFontsVersion("2")
 	}
-	presentation.SetCustomBranches(userConfig.Gui.BranchColors)
+
+	if userConfig.Gui.BranchColorPatterns != nil {
```

Comment:
It doesn't seem to make a difference in practice, but I'd find it clearer to say `len(userConfig.Gui.BranchColorPatterns) > 0`.